examples/http-to-websocket-upgrade/http_to_websocket_upgrade.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how an HTTP endpoint is updated to a WebSocket endpoint using an upgrade resource.
```

Comment:
Edit: This BBE demonstrates how an HTTP endpoint is updated to a WebSocket endpoint using an upgrade resource.